src/screens/NestedInfoScreen.tsx

```diff
@@ -42,6 +57,20 @@ const ListHeaderComponent = ({ loading, html }: { loading: boolean; html?: strin
       <Wrapper>
         {/* @ts-expect-error HtmlView uses memo in js, which is not inferred correctly */}
         <HtmlView html={html} />
+        {!!subQuery && !!subQuery.routeName && subQuery.params && (
+          <Button
+            title={subQuery.buttonTitle}
+            onPress={() =>
+              navigation.navigate({
+                name: subQuery.routeName,
+                params: {
+                  ...subQuery.params,
+                  title: subQuery.params?.title
```

Comment:
i missed here. `params: subQuery.params` is more likely.

Reply:
why have you decided to destruct the params of subquery here now instead of setting `params: subQuery.params` as is suggested? what do you think is the benefit?

Reply:
The code has been edited.
https://github.com/ikuseiGmbH/smart-village-app-app/commit/13f58939e0a9c13d144000a6c7285c32e8d4a385